cpp/references/data_structures/list_digraph.cpp

```diff
@@ -1 +1,288 @@
-;
\ No newline at end of file
+#include "list_digraph.hpp"
+
+using namespace godot;
+
+
+/* ########################### VERTEX ########################### */
+
+typedef ListDigraph::Vertex Vertex;
+
+/* ##### GDNATIVE API ##### */
+
+Vertex *Vertex::_new(lemon::ListDigraph::Node p_vertex, ListDigraph *p_graph) {
+    Vertex* vertex = _new();
+    vertex->_vertex = p_vertex;
+    vertex->_graph = p_graph;
+    return vertex;
+}
+
+lemon::ListDigraph::Node Vertex::get_vertex() const {
+    return _vertex;
+}
+
+/* ##### GDSCRIPT API ##### */
```

Comment:
Because MultiScript is not yet supported in Godot, and because users may want to extend GraphVertex / GraphArc with GDScript classes, I think we may want to consider also allowing users to define script properties in ListDigraph that can GraphVertex and GraphArc. Then we can use `_get`, `_set`, and `_get_property_list` to simulate that the properties are accessible through the GraphVertex and GraphArc. We can then also provide a `get_wrapper()` method or something to provide users with the instance of their scripted type that has a reference to the GraphVertex/GraphArc (so they can mess with the `data` property directly from their script).